app/assets/stylesheets/elements/_buttons.css.scss

```diff
@@ -0,0 +1,40 @@
+// Dash Panel Buttons
+.nav-pills > li > a{
+  color: $text-white;
+  background-color: $dark-gray;
+  width: 150px;
+  height: 49px;
+  text-align: center;
+  border-radius:0;
+
+  p {
+    width: 108px;
+    height: 20px;
+    font-family: $default-font-bold;
+    font-size: 15px;
+  }
+}
+
+.nav-pills > li.active > a,
+.nav-pills > li.active > a:hover,
+.nav-pills > li.active > a:focus {
+    color: $text-light;
+    background-color: $light-gray;
+    width: 150px;
+    height: 49px;
+}
+
+// Login Button
+a.btn-login {
+  display: block;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 22px;
+  text-decoration: none;
+  font-size: 18px;
```

Comment:
We should make all of our common font styles mixins and just include them when we need them. That way our typography stays consistent.